.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -3,13 +3,18 @@ name: Pull request
 about: Tell us about your contribution
 ---
 
-<!--  Thanks for contributing to Contour!
+---
+## Thanks for contributing to Contour!
+
+Please delete this text between the lines (and the lines) before submitting your PR.
```

Comment:
Between which lines? Is the separator missing?

Reply:
The `---` characters will create a HR when rendered, but I'll make this more clear, thanks.